functions/src/update-contract-metrics.ts

```diff
@@ -0,0 +1,98 @@
+import * as functions from 'firebase-functions'
+import * as admin from 'firebase-admin'
+import { groupBy, sortBy } from 'lodash'
+
+import { getValues, invokeFunction, log, writeAsync } from './utils'
+import { Bet } from '../../common/bet'
+import { Contract, CPMM } from '../../common/contract'
+import { DAY_MS } from '../../common/util/time'
+import {
+  calculateProbChanges,
+  computeElasticity,
+  computeVolume,
+} from '../../common/calculate-metrics'
+import { getProbability } from '../../common/calculate'
+import { batchedWaitAll } from '../../common/util/promise'
+import { newEndpointNoAuth } from './api'
+
+const firestore = admin.firestore()
+export const scheduleUpdateContractMetrics = functions.pubsub
+  .schedule('every 15 minutes')
+  .onRun(async () => {
+    try {
+      console.log(await invokeFunction('updatecontractmetrics'))
+    } catch (e) {
+      console.error(e)
+    }
+  })
```

Comment:
What do you think about the idea of having one scheduler function that calls all three of the update cloud functions?

It's nice that currently the scheduled function is co-located, but it might be simpler overall to have one scheduler process. I guess I could go either way here.